source/_includes/common-tasks/enable_i2c.md

```diff
@@ -62,3 +62,28 @@ You can enable i2c via this terminal:
   sync
   reboot
   ```
+#### Check I2C activation on OS level
+
+After the reboot of the host, the I2C activation should now be visible under /dev. You may check by using the add-on `Terminal & SSH` and list the directory `/dev`. If `ls` shows an entry `i2c-1` or similar, your I2C config might work. You can check the status of I2C kernel modules by entering `lsmod | grep i2c`. If they are loaded, you should find at least the entry `i2c_dev`. Active usage of the modules is indicated by a number, e.g. `i2c_dev 20480 2` would indicate two active I2C data sources (e.g. a BMP280 temperature & pressure sensor) 
+
+An active I2C can also be check with a multi meter showing 3.3 V on the I2C pins GPIO2 and GPIO3. 
```

Comment:
This section does not apply to a regular installation and requires third party add-ons. It should therefore not be added here.

Reply:
Removed the reference to third party add on. Kept lsmod and more explicit description of the dev folder